pandas/tests/arrays/categorical/test_indexing.py

```diff
@@ -122,6 +162,60 @@ def test_get_indexer_non_unique(self, idx_values, key_values, key_class):
             tm.assert_numpy_array_equal(expected, result)
             tm.assert_numpy_array_equal(exp_miss, res_miss)
 
+    def test_where_unobserved_nan(self):
```

Comment:
is where all of the where tests are?

Reply:
There weren't any previously (we used to fall back to object).